### PR TITLE
Remove duplicate if conditions

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/io/exporttoken/WeaponToken.java [hunk 1]
```diff
@@ -1887,8 +1887,8 @@ else if (hitMode == HITMODE_OHHIT)
 				baseBonus += pc.getOffHandLightBonus();
 			}
 
-			if ((hitMode == HITMODE_TWOHIT) || (hitMode == HITMODE_OHHIT) || (hitMode == HITMODE_TWFOHL)
-				|| (hitMode == HITMODE_TWFOHH))
+			if ((hitMode == HITMODE_OHHIT) || (hitMode == HITMODE_TWFOHL)
+					|| (hitMode == HITMODE_TWFOHH))
```
Comment (thpr) [inline code/src/java/pcgen/io/exporttoken/WeaponToken.java]: Doesn't this strike you more as a bug than a potential optimization?

Comment (karianna) [reply code/src/java/pcgen/io/exporttoken/WeaponToken.java]: I don't see the duplicate either?

Comment (grimreaper) [reply code/src/java/pcgen/io/exporttoken/WeaponToken.java]: I don't think its a bug, but you're right that this is weird. better fix applied.

File: code/src/java/pcgen/io/exporttoken/WeaponToken.java [hunk 2]
```diff
@@ -1550,7 +1548,7 @@ else if (display.isSecondaryWeapon(eq) && display.hasPrimaryWeapons())
 			else
 			{
 				// offhand heavy
-				hitMode = HITMODE_TWFOHH;
+				hitMode = HITMODE_TWOHIT;
```
Comment (LegacyKing) [inline code/src/java/pcgen/io/exporttoken/WeaponToken.java]: Looks like we have logic changes here.  We had Two Weapon Fighting Off Hand Heavy {TWFOHH} changed to TWOHIT.

**Comment by karianna** [comment]
So basically even though the two constants have the same value, it's still useful to have both for readability..

**Comment by LegacyKing** [comment]
It looks like we're losing an entire HitCategory (We have 4 conditions dropped to 3). I don't feel good about this change. For some reason we've completely dumped the Two Weapon Off Hand Heavy group.  What exactly is this supposed to be fixing or addressing? (Mind you, EQUIPMENT is one of the most heavily magic'd code in pcgen that I'm aware of, and these values are displayed on the character sheet).

**Comment by karianna** [comment]
I'm actually going to close this PR.

**Pull request closed without merging**